barretenberg_static_lib/src/composer.rs

```diff
@@ -227,26 +227,10 @@ mod test {
             public_inputs: None,
             result: false,
         };
-        // This should fail as we specified that we do not have any public inputs
```

Comment:
Why did we get rid of these tests?

Reply:
The proving system's behaviour has changed -- If you specify that there will be 2 public inputs and you do not pass two public inputs, then it will panic or abort if it is using wasm